internal/pkg/grpcsupport/grpcservicegen.go

```diff
@@ -0,0 +1,350 @@
+package grpcsupport
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/TIBCOSoftware/flogo-lib/logger"
+)
+
+const (
+	serviceName = "\nservice "
+	grpcGenPath = "github.com/TIBCOSoftware/mashling/gen/grpc"
+)
+
+//Logger
+var log = logger.GetLogger("mashling-cli-grpc-gen")
```

Comment:
Should we be using the Flogo logging mechanism for CLI/non-runtime behaviors?

Reply:
Agreed. Will change this to go logs.